src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5173,6 +5175,89 @@ namespace HeatBalanceManager {
+        if (state.dataSurface->TotSurfIncSolMultiplier > 0) {
+            if (!allocated(state.dataSurface->SurfIncSolMultiplier)) {
+                // could be extended to interior surfaces later
+                state.dataSurface->SurfIncSolMultiplier.allocate(state.dataSurface->TotSurfaces);
+            }
+
+            int NumAlpha;
+            int NumNumeric;
+            int IOStat;
+            for (int Loop = 1; Loop <= state.dataSurface->TotSurfIncSolMultiplier; ++Loop) {
+                state.dataInputProcessing->inputProcessor->getObjectItem(state,
+                                                                         cCurrentModuleObject,
+                                                                         Loop,
+                                                                         state.dataIPShortCut->cAlphaArgs,
+                                                                         NumAlpha,
+                                                                         state.dataIPShortCut->rNumericArgs,
+                                                                         NumNumeric,
+                                                                         IOStat,
+                                                                         state.dataIPShortCut->lNumericFieldBlanks,
+                                                                         state.dataIPShortCut->lAlphaFieldBlanks,
+                                                                         state.dataIPShortCut->cAlphaFieldNames,
+                                                                         state.dataIPShortCut->cNumericFieldNames);
+                if (UtilityRoutines::IsNameEmpty(state, state.dataIPShortCut->cAlphaArgs(1), cCurrentModuleObject, ErrorsFound)) {
+                    ShowContinueError(
+                        state, "...each SurfaceProperty:SolarIncidentInside name must not duplicate other SurfaceProperty:SolarIncidentInside name");
+                    continue;
+                }
+
+                // Assign surface number
+                int SurfNum = UtilityRoutines::FindItemInList(state.dataIPShortCut->cAlphaArgs(1), state.dataSurface->Surface);
+                if (SurfNum == 0) {
+                    ShowSevereError(state,
+                                    std::string{RoutineName} + cCurrentModuleObject + "=\"" + state.dataIPShortCut->cAlphaArgs(1) +
+                                        ", object. Illegal value for " + state.dataIPShortCut->cAlphaFieldNames(1) + " has been found.");
+                    ShowContinueError(state,
+                                      state.dataIPShortCut->cAlphaFieldNames(1) + " entered value = \"" + state.dataIPShortCut->cAlphaArgs(1) +
+                                          "\" no corresponding surface (ref BuildingSurface:Detailed) has been found in the input file.");
+                    ErrorsFound = true;
```

Comment:
You can similarly do a `continue;` here (as you do in the block above) which eliminates the `else` and un-indents the rest of the loop body.